dashboard/app/helpers/levels_helper.rb

```diff
@@ -248,6 +248,10 @@ def app_options
         view_options.camelize_keys
       end
 
+    if @script_level && @level.can_have_feedback?
+      @app_options[:serverScriptLevelId] = @script_level.id
```

Comment:
to be honest I am not quite clear on whether this belongs directly on app_options or whether it should be added via `view_options` like `serverLevelId` these here: https://github.com/code-dot-org/code-dot-org/blob/1f16b2347fbc29cd2631c674bf8e71229ed3b6f7/dashboard/app/helpers/levels_helper.rb#L171-L177

It appears that `view_options` is read from within rails views whereas app options is only read by the client. so as long as you are only accessing this from the client, then it seems fine to add it directly to app options, like you are currently doing.

Reply:
Yeah, I was wondering about this too. Thanks for explaining the distinction.  I only need access to the `server_script_level_id` on the client in a React component, `TeacherFeedback`, so I think it's ok here.